pkg/arvo/app/chat-cli.hoon

```diff
@@ -613,18 +622,22 @@
       =-  [[- moz] this]
       %^  act  %do-create  %chat-view
       :-  %chat-view-action
-      :^  %create  path  security
-      ::  ensure we can read from/write to our own chats
-      ::
-      :-  ::  read
+      :*  %create
+          path
+          security
+          ::  ensure we can read from/write to our own chats
+          ::
+          ::  read
           ?-  security
             ?(%channel %journal)  ~
             ?(%village %mailbox)  [our-self ~ ~]
           ==
-      ::  write
-      ?-  security
-        ?(%channel %mailbox)  ~
-        ?(%village %journal)  [our-self ~ ~]
+          ::  write
+          ?-  security
+            ?(%channel %mailbox)  ~
+            ?(%village %journal)  [our-self ~ ~]
+          ==
+          ?~(allow-history %.y u.allow-history)
```

Comment:
`+fall` exists. But if we're gonna do defaulting to constants, probably do that in the parser instead of in the logic. Same for `;join`.

Reply:
cool, will use +fall

Reply:
done